specification/cosmos-db/data-plane/Microsoft.TablesStorage/preview/2018-10-10/table.json

```diff
@@ -0,0 +1,1585 @@
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success.",
+            "headers": {
+              "x-ms-client-request-id": {
+                "x-ms-client-name": "ClientRequestId",
+                "type": "string",
+                "description": "If a client request id header is sent in the request, this header will be present in the response with the same value."
+              },
+              "x-ms-request-id": {
+                "x-ms-client-name": "RequestId",
+                "type": "string",
+                "description": "This header uniquely identifies the request that was made and can be used for troubleshooting the request."
+              },
+              "x-ms-version": {
+                "x-ms-client-name": "Version",
+                "type": "string",
+                "description": "Indicates the version of the Table service used to execute the request. This header is returned for requests made against version 2009-09-19 and above."
+              },
+              "Date": {
+                "type": "string",
+                "format": "date-time-rfc1123",
+                "description": "UTC date/time value generated by the service that indicates the time at which the response was initiated"
+              }
+            },
+            "schema": {
+              "$ref": "#/definitions/StorageServiceStats"
+            }
+          },
+          "default": {
+            "description": "Failure",
+            "headers": {
+              "x-ms-error-code": {
+                "x-ms-client-name": "ErrorCode",
+                "type": "string"
+              }
+            },
+            "schema": {
+              "$ref": "#/definitions/StorageError"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "StorageError": {
+      "type": "object",
+      "properties": {
+        "Message": {
+          "type": "string"
+        }
+      }
+    },
+    "StorageServiceProperties": {
+      "description": "Storage Service Properties.",
+      "type": "object",
+      "properties": {
+        "Logging": {
+          "description": "Azure Analytics Logging settings",
+          "$ref": "#/definitions/Logging"
+        },
+        "HourMetrics": {
+          "description": "A summary of request statistics grouped by API in hourly aggregates for queues",
+          "$ref": "#/definitions/Metrics"
+        },
+        "MinuteMetrics": {
+          "description": "a summary of request statistics grouped by API in minute aggregates for queues",
+          "$ref": "#/definitions/Metrics"
+        },
+        "Cors": {
+          "description": "The set of CORS rules.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/CorsRule",
+            "xml": {
+              "name": "CorsRule"
+            }
+          },
+          "xml": {
+            "wrapped": true
+          }
+        }
+      }
+    },
+    "Logging": {
```

Comment:
This name triggers 
`Error AZC0012	Single word class names are too generic and have high chance of collision with BCL types or types from other libraries`

Could we rename it to `LoggingSettings` ?